dmscripts/notify_buyers_to_award_closed_briefs.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, date, timedelta
+
+import dmapiclient
+from dmutils.email import DMNotifyClient
+from dmutils.email.exceptions import EmailError
+from dmutils.formats import DATE_FORMAT
+
+from dmscripts.helpers import logging_helpers, date_helpers, brief_data_helpers
+
+logger = logging_helpers.configure_logger({'dmapiclient': logging_helpers.logging.INFO})
+
+
+def _filter_brief_users_by_optional_user_id_list(brief, user_id_list):
+    """Filter the brief's users by restricted user ID list if present"""
+    if user_id_list:
+        return filter(lambda user: user['id'] in user_id_list, brief['users'])
+    else:
+        return brief['users']
+
+
+def _filter_briefs_by_user_id_list(briefs, user_id_list):
+    """Return a list of briefs if they have at least 1 user in the given ID list"""
+    return [
+        brief for brief in briefs if any(
+            _filter_brief_users_by_optional_user_id_list(brief, user_id_list)
+        )
+    ]
+
+
+def _create_context_for_brief(brief):
+    return {
+        'brief_id': brief['id'],
+        'brief_title': brief['title'],
+        'utm_date': date.today().strftime("%Y%m%d")
+    }
+
+
+def notify_users(notify_client, notify_template_id, brief, user_id_list):
+    failed_users = []
+    if brief['users']:
+        context_data = _create_context_for_brief(brief)
+        brief_users = _filter_brief_users_by_optional_user_id_list(brief, user_id_list)
+
+        for user in brief_users:
+            if user['active']:
+                try:
+                    notify_client.send_email(
+                        user['emailAddress'], notify_template_id, context_data, allow_resend=False
+                    )
+                except EmailError:
+                    failed_users.append(user['id'])
+
```

Comment:
Could use a set here because I don't think we care how many times the user appears but just an observation. Fine either way